### PR TITLE
add manifest druid and @id url to canvas indexing and json - closes #849

### LINKED ISSUE #849: Attach a manifest link to the Canvas model.
- [x] choose content type for canvas resources
- [x] provide access to parent manuscript druid (fixed by #851)
- [x] build or pass custom manifest id through to canvas concern

### PR DESCRIPTION
**Pull request by aeschylus**

### CODE REVIEW

**Comment by jkeck** [comment]
LGTM, I would like somebody else to review since we paired on most of this together.

File: app/models/iiif_canvas_indexer.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@ def index_canvases
         enhanced_canvas = JSON.parse(canvas.to_json)
         enhanced_canvas['manifest_label'] = manifest.label
         enhanced_canvas['parent_manuscript_number'] = document.fetch('manuscript_number_tesim', nil)
+        enhanced_canvas['parent_druid'] = document.id
```
Comment (mejackreed) [inline app/models/iiif_canvas_indexer.rb]: Is this meant to represent the druid of the "manuscript" (SDR object) ? Is it similar to this? 

https://github.com/sul-dlss/exhibits/commit/84b57aaa49c7c129c8b1a14cf9720a61f3ccc110

Comment (jkeck) [reply app/models/iiif_canvas_indexer.rb]: Ah, we didn't catch that in the resulting solr document.  Yeah, that's what it was intended to be, so we can probably just re-use that field.

**Comment by aeschylus** [comment]
@mejackreed Should be fixed up now.

**Pull request merged**